spacy_llm/tasks/builtin_task.py

```diff
@@ -0,0 +1,324 @@
+import abc
+from pathlib import Path
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, Type, cast
+
+import jinja2
+import srsly
+from spacy import Language, util
+from spacy.tokens import Doc
+from spacy.training import Example
+
+from ..registry import lowercase_normalizer
+from ..ty import FewshotExample, TaskResponseParser
+
+
+class BuiltinTask(abc.ABC):
+    """Abstract base task implementing interfaces and/or functionality expected from all built-in tasks:
+        - working prompt template strings
+        - swappable response parsers
+        - swappable prompt example type
+        - integration of fewshot example into the fully rendered prompt
+        - initializable (in line with other spaCy components)
+        - (de-)serialization
+
+    On the relation of BuiltinTask to LLMTaskProtocol: the latter specifies the minimal contract a task implementation
```

Comment:
ctrl-F and replace:
- LLMTaskProtocol
- TaskResponseParserProtocol

Reply:
Fixed in https://github.com/explosion/spacy-llm/pull/242/commits/b687d54a5b3f72dfd05f848e490c140b45d70a3b.